tests/Feature/Platform/Actions/WriteGameSetSortTitleActionTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Platform\Actions;
+
+use App\Models\GameSet;
+use App\Platform\Actions\WriteGameSetSortTitleAction;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class WriteGameSetSortTitleActionTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testItGeneratesCorrectSortTitles(): void
+    {
+        // Arrange
+        $gameSet = GameSet::factory()->create(['title' => '[Series - The Legend of Zelda]', 'sort_title' => null]);
+
+        // Act
+        (new WriteGameSetSortTitleAction())->execute($gameSet, $gameSet->title);
+        $gameSet->refresh();
+
+        // Assert
+        $this->assertEquals('series - the legend of zelda', $gameSet->sort_title);
```

Comment:
Isn't the whole point of this feature to eliminate the "the"?

I guess the user could remove it themselves.

Reply:
It'd probably be best if we added a thin layer of preprocessing to these. I've done that in latest and also updated this test case.